homework9/js/main.js

```diff
@@ -0,0 +1,134 @@
+//Створити функцію-конструктор Людини. Властивості, які будуть описувати екземпляр
+// Людини - ім'я, вік, стать, національність, країна, список країн для подорожей.
+// Створити універсальні функції - знайомство, прокидатися, засинати й список країн, які
+// Людина хоче відвідати. Роботу цих функцій можна виводити через console.log.
+// Але в кожній з функцій повинно використовуватись якомога більше характеристик конкретної
+// Людини на якій ця функція викликається (підказка: тут треба використати call/apply)
+function People(name, age, sex, nationality, country, listCountry){
+    this.name = name;
+    this.age = age;
+    this.nationality = nationality;
+    this.country = country;
+    this.listCountry = listCountry;
+}
+
+const hello = function () {
+    console.log(`Hello, I'm ${this.name}. I'm ${this.age} years old. In addition, I'm ${this.nationality}.\nNice to meet you!`);
+}
+
+const wakeup = function (){
+    console.log(`Good morning! ${this.name} wake up now!`);
+}
+
+const sleep = function (){
+    console.log(`Dear ${this.nationality}, you must sleep at 11 p.m. o'clock.`);
+}
+
+const list = function (){
+
+    console.log(`I'm from ${this.country} and I want visit ${this.listCountry}`);
+}
+
+let person = new People('Alise', 28, `female`, `ukrainian`, `Ukraine`, ['Korea', 'Germany', 'Italy']);
+
+hello.bind(person)(person.name, person.age, person.nationality);
+wakeup.apply(person, [person.name]);
+sleep.apply(person, [person.nationality]);
+list.call(person, [person.country, person.listCountry]);
+
+
+//Створити власну реалізацію методу .bind (підказка: в кінці треба помістити цю власну функцію у прототип -
+// Function.prototype.myOwnBind = function () { [тут_код_кастомного_bind] }
+
+const cat = {
+    name: "Alice",
+    breed: "maine coon",
+}
+
+function description(name, breed) {
+    console.log(`Cat name is ${this.name}. It is ${this.breed}.`);
+}
+
+myOwnBind = function (func, context) {
+    return func.apply(context);
+}
+
+myOwnBind(description, cat);
+
+
+//Cтворити функцію, котра приймає 2 параметри - об'єкти.
+// Функція повинна перевіряти чи ці 2 обʼєкти абсолютно ідентичні та
+// повертає результат у зрозумілому форматі
+const dog1 = {
+    breed: "buldog",
+    name: "Rex",
+};
+
+const dog2 = {
+    breed: "buldog",
+    name: "Rex",
+};
+
+const dog3 ={
+    breed: "buldog",
+    name: "Monica"
+}
+
+function checkObject(object1, object2){
```

Comment:
Please use more meaningful names.

It works for nested objects too, nice 👍 
<img width="540" alt="image" src="https://user-images.githubusercontent.com/84010501/194717241-2a9ea5da-4c4b-49f1-b192-7234161b221c.png">